src/orbit.rs

```diff
@@ -67,6 +71,13 @@ where
     Ok(SimpleOrbit { ipfs, oid, policy })
 }
 
+pub fn verify_oid_v0(oid: &Cid, pkh: &str, salt: &str) -> bool {
+    match Code::try_from(oid.hash().code()) {
+        Ok(code) => &code.digest(format!("{}:{}", salt, pkh).as_bytes()) == oid.hash(),
```

Comment:
Why can't we trust that the hash is correct? It feels to me like we could have this guarantee at creation time or something, instead of recreating the hash. But I might be too naive.

Reply:
creation-time occurs on the client which may not be trustworthy. The orbit ID is included in the signed creation message so that there's a solid link between the message and the subject-orbit (imagine searching for the creation message for an orbit if the orbit-id was not included). We could just accept the info used to generate the orbit ID and do it on the server, but we'd still be creating a hash anyway